addons/mail/security/mail_security.xml

```diff
@@ -27,6 +27,20 @@
             <field name="domain_force">[(1, '=', 1)]</field>
         </record>
 
+        <record id="discuss_gif_favorite_rule" model="ir.rule">
+            <field name="name">Discuss GIF favorite</field>
+            <field name="model_id" ref="model_discuss_gif_favorite"/>
+            <field name="groups" eval="[Command.link(ref('base.group_user'))]"/>
+            <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        </record>
+
+        <record id="discuss_gif_favorite_admin" model="ir.rule">
+            <field name="name">Mail.gif.favorite: admin full access</field>
```

Comment:
Inconsistent naming between the 2 rules

- `discuss` or `mail`? (-> `discuss`?)
- dot or phrase format? (combination of both like other rules?)

Reply:
Let's go for Discuss